messWebsite/settings.py

```diff
@@ -4,7 +4,7 @@
 import environ
 
 env = environ.Env()
-environ.Env.read_env()
+environ.Env.read_env(Path(__file__).resolve().parent.parent / '.env')
```

Comment:
we should do it at line 11 after `BASE_DIR`. as `BASE_DIR` is having the same value